cli/ballerina-cli/src/test/resources/test-resources/buildToolResources/tools/sample-build-tool-pkg/BalTool.toml

```diff
@@ -0,0 +1,5 @@
+[tool]
+id="sample-build-tool"
+
+[[dependency]]
+path="./resources/sample-build-tool-1.0.0.jar"
```

Comment:
```suggestion
path="./resources/sample-build-tool-1.0.0.jar"

```

Reply:
Addressed in [0051112](https://github.com/ballerina-platform/ballerina-lang/pull/44104/commits/0051112995b8aa8fd9bb7df0099b63fe94e3a5b1)